spec/support/aws/get_metric_statistics_response.rb

```diff
@@ -0,0 +1,2 @@
+GetMetricStatisticsDatapoint = Struct.new(:timestamp, :sum)
```

Comment:
WDYT of moving this to its own file to make it easier to find? match rails autoloading conventions

Reply:
No strong feelings, happy to move it.